docs/infrastructure/eks-quickstart.md

```diff
@@ -1,204 +1,159 @@
-They can usually be easily installed using your package manager. For instance, on
-macOS, if you are using [Homebrew](https://brew.sh), you can just run:
+    ```bash
+    git clone https://github.com/codacy/chart.git
+    cd chart/docs/infrastructure/EKS/
+    ```
 
-```bash
-brew install awscli terraform kubernetes-helm kubectl
-```
+    The folder includes the following infrastructure stacks:
 
-You'll also need to setup the CLI credentials for your AWS account. See how
-to do it the [AWS](https://docs.aws.amazon.com/polly/latest/dg/setup-aws-cli.html)
-and [Terraform](https://www.terraform.io/docs/providers/aws/index.html) documentation.
-Note that, as stated in the documentation, if your `.aws/credentials` are fairly
-complex you might need to set `AWS_SDK_LOAD_CONFIG=1` for Terraform to work correctly.
+    * **backend**: Optional S3 bucket for storing the Terraform state and the DynamoDB table for state locking
+    * **main**: Amazon EKS cluster, including the setup of all network and node infrastructure to go from zero to a fully functional cluster
+    * **setup**: Additional setup to be performed before installing Codacy on your vanilla Amazon EKS cluster
 
-## TL;DR
+## 2. Set up the Terraform state storage backend
 
-### Setting up an EKS cluster for Codacy
+The [backend](https://www.terraform.io/docs/backends/index.html) stores the current and historical state of your infrastructure.
 
-Assuming **AWS is well configured** and you fulfill all the prerequisites,
-the setup without state storage, can be done with:
+Although using the backend is optional, we recommend that you deploy it, particularly if you are planning to use these templates to make modifications to the cluster in the future:
 
-```bash
-$ export AWS_SDK_LOAD_CONFIG=1
-$ cd main/
-$ terraform init && terraform apply # should take around 15 mins
-$ cd ../setup/
-$ terraform init && terraform apply
-$ aws eks update-kubeconfig --name codacy-cluster
-```
+1. Initialize Terraform and apply the changes inside the `backend/` directory, then follow Terraform's instructions:
 
-## Deployment - Long version
+    ```bash
+    cd backend/
+    terraform init && terraform apply
+    ```
 
-### 1. `backend` - setup terraform state storage
+    An Amazon S3 bucket with a unique name to save the infrastructure state is created.
 
-The [backend](https://www.terraform.io/docs/backends/index.html) stores
-the current and historical state of your infrastructure. Deploying and using
-it is optional, but it might make your life easier if you are planning to use
-these templates to make modifications to the cluster in the future.
+1. Note the value of `state_bucket_name` in the output of the command.
 
-To deploy it run:
+1. Edit the `config.tf` files that exist in the `main/` and `setup/` directories and follow the instructions to set the name of the Amazon S3 bucket and to enable the use of the backend in those files.
```

Comment:
Maybe change `in those files` to `in those infrastructure stacks` to make the effect of using a backend clearer.

Reply:
Makes sense!